### ISSUE TITLE
Aggregator broken on state.events

### ISSUE THREAD

**Issue by obfuscurity** [graphite-project/carbon#350]
As reported by @jraby in https://github.com/graphite-project/carbon/pull/308#issuecomment-68738847, he's seeing the following errors since https://github.com/graphite-project/carbon/commit/26b7484bc8a3aa2e464035c739f1c83efebffa14:

```
05/01/2015 11:50:31 :: Unhandled error in Deferred:
05/01/2015 11:50:31 :: Unhandled Error
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/base.py", line 1169, in run
    self.mainLoop()
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/base.py", line 1178, in mainLoop
    self.runUntilCurrent()
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/base.py", line 800, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/task.py", line 215, in __call__
    d = defer.maybeDeferred(self.f, *self.a, **self.kw)
--- <exception caught here> ---
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 134, in maybeDeferred
    result = f(*args, **kw)
  File "/opt/graphite/lib/carbon/aggregator/buffers.py", line 66, in compute_value
    state.events.metricGenerated(self.metric_path, datapoint)
exceptions.AttributeError: 'module' object has no attribute 'metricGenerated'
```

/cc @jssjr

**Comment by jraby** [comment]
I have a fix for this coming in the next minutes.

**Comment by obfuscurity** [comment]
Should be fixed now via #351.
